docs/contributing/technical/resolve-merge-conflicts.md

```diff
@@ -70,9 +69,9 @@ You will see something similar to:
 
 ![proper merge but results in conflicts](../../../static/img/contributing-resolve-merge-conflicts-merge-conflicts.png)
 
-## Review changes
+## Review Changes
```

Comment:
We might want to add how to see the difference of the conflicts on VS Code here.

Reply:
That would make a great blog post to link here. What do you think?

Reply:
I think that'll work 😊